tests/svg_colr_svg_test.py

```diff
@@ -46,6 +46,7 @@ def test_svg_to_colr_to_svg(svg_in, expected_svg_out, config_overrides):
         (svg_in,),
     )
     _, ttfont = write_font._generate_color_font(config, glyph_inputs)
+    ttfont.saveXML("-")  # TEMPORARY
```

Comment:
remove if TEMPORARY, or just keep if useful for debugging (pytest will capture stdout by default unless assertion fails)

Reply:
You were getting good at spotting my prints so I had to start finding new ways to sneak useless console noise past